warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -958,18 +1091,21 @@ models:
           that is being summarized here, and counts may be inflated due to multiple rows with identical identifiers.
   - name: fct_daily_scheduled_stops
     description: |
-      A daily table showing all stops that had trip stop events scheduled on a given date.
+      Each row is a summary of trip activity by stop for a given feed on a given agency-defined `service_date`.
+
+      A stop here is defined within an individual feed so, even if a given stop is served by multiple
+      agencies, the stop will have one row per feed in which it appears.
```

Comment:
this is a bit unclear; maybe remove `even if` and the reference to agencies?

Reply:
The point here is that the same physical location may appear multiple times in the table if it appears in multiple feeds; we are not doing stop entity resolution across feeds (this is a thorny topic in GTFS world.) I can try to clarify but I think that it's important to note that in some form.

Reply:
https://github.com/cal-itp/data-infra/pull/2457/commits/fba7014673f92f9016c806b0d44e4125fcfe5f00